packages/terra-application-layout/src/terra-dev-site/doc/common/ApplicationMenu.scss

```diff
@@ -0,0 +1,12 @@
+@import './ApplicationLayoutDocCommon';
+
+:local {
+  .footer {
+    @extend %common-container-content;
```

Comment:
Lets directly add the styles that we are using here instead of using the @extends syntax. I think this add extra complexity to these examples that aren't needed.

Reply:
Done